.github/workflows/deploy.yaml

```diff
@@ -3,7 +3,8 @@ on:
   push:
     branches:
       - main
-      - add-foundations # test the deploy
+      - dev
+      - dev-deploy
```

Comment:
Awesome! So to rebuild the dev server/package/image, one should push to either one of `dev` or `dev-deploy`?

Reply:
Should be just `dev`, I'll remove dev-deploy :)